creator.cpp

```diff
@@ -21,6 +21,7 @@
 #include "ui_creator.h"
 #include "version.h"
 
+#include <QtCore/qregularexpression.h>
```

Comment:
```suggestion
#include <QRegularExpression>
```

Reply:
Actually this can be removed as I used `core5compat` instead.

Reply:
Done